### ISSUE TITLE
Add an exception handler for AccountLockedException

### ISSUE THREAD

**Issue by sadatmalik** [talentbeyondboundaries/tbbtalentv2#13]
javax account locked exceptions thrown by the user service are passed through the external api - add a handler to convert these to a service exception as appropriate

**Comment by ajt001** [comment]
Won't this be fixed by implementing new Exception classes extending ServiceException as outlined in #12?

**Comment by sadatmalik** [comment]
Yes, if we standardise on extending ServiceException; then this ticket will no longer be needed.
